rootfs/scheduler/base.py

```diff
@@ -0,0 +1,8 @@
+from django.conf import settings
+from .client import KubeHTTPClient
+
+
+class Base(object):
```

Comment:
Ingesting `settings` directly here - we were anyway doing that in a different part of the code instead of using what's passed in from the model but at the end of the day we may want to abstract the passing out.